torchao/quantization/quant_api.py

```diff
@@ -141,11 +149,11 @@ def change_linear_weights_to_int8_dqtensors(model, filter_fn=None):
         )
 
     _replace_with_custom_fn_if_matches_filter(
-        model, _get_subclass_inserter(Int8DynamicallyQuantizedLinearWeight), filter_fn
+        model, _get_subclass_inserter(Int8DynamicallyQuantizedLinearWeight, enable_parametrization=True, **kwargs), filter_fn
```

Comment:
Maybe the default value for `enable_parametrization` can be `TORCH_VERSION_AFTER_2_4`?

Reply:
OK sure